### ISSUE TITLE
Self-reference doesn't match document location when validating feed

### ISSUE THREAD

**Issue by lowescott** [jekyll/jekyll-feed#93]
Hi all, I'm a new user to jekyll-feed, and I suspect the behavior I'm seeing is simply due to a misconfiguration on my part. However, I'm having a hard time finding the misconfiguration. When validating the Atom feed (you can see the validation results at http://feedvalidator.org/check?url=http:%2F%2Fblog.scottlowe.org%2Ffeed.xml) it reports "self-reference doesn't match document location", and I note that it's reporting the document at "http://blog.scottlowe.org//feed.xml". Any suggestions on how I can fix this to report the correct document location? Note that the plugin _is_ generating a valid feed at http://blog.scottlowe.org/feed.xml, just the location is being reported incorrectly.

**Comment by pathawks** [comment]
In your `_config.yml`, you set [`baseurl` to `/`](https://github.com/lowescott/lowescott.github.io/blob/master/_config.yml#L16).

[Don't do that](https://byparker.com/blog/2014/clearing-up-confusion-around-baseurl/).

> **TL;DR: Use baseurl when you are building a site that doesn’t sit at the root of the domain.**

**Comment by lowescott** [comment]
@pathawks Thanks. I included `baseurl` because I migrated to GitHub Pages from WordPress and was having difficulties maintaining the 1500+ permalinks from the previous site. I seem to recall at the time that I couldn't make it work properly without setting `baseurl`, but I'll try it again and see what (if anything) it breaks.

**Comment by pathawks** [comment]
If anything does break, please ping me on [talk.jekyllrb.com](https://talk.jekyllrb.com/) and we can dig in deeper.

`baseurl` caused me a fair number of headaches when I migrated my personal blog to Jekyll as well.
